torch/_inductor/compile_fx.py

```diff
@@ -918,11 +918,17 @@ def log_graph_runnable() -> str:
                         colored=True,
                     ),
                 )
+                provenance_tracking_json = torch.fx.traceback.get_graph_provenance_json(
+                    gm.graph
+                )
+                post_grad_graphs_log.debug("%s", provenance_tracking_json)
                 trace_structured(
                     "inductor_post_grad_graph",
                     payload_fn=lambda: gm.print_readable(
                         print_output=False, include_stride=True, include_device=True
-                    ),
+                    )
+                    + "# "
+                    + provenance_tracking_json,
```

Comment:
I am not sure about the answer, but will changing the log here confuse tlparse?

Reply:
I've tried it on tlparse, and tlparse will just have this additional comment line in the resulting post_grad_graph graph file. 

> I am not sure about the answer, but will changing the log here confuse tlparse?